### PR TITLE
add more wasm host functions and support `wasm_init`

### PR DESCRIPTION
**Pull request by localvar**

### CODE REVIEW

**Review by megaeasex — CHANGES_REQUESTED** [review]
[TASK:easegress-pr-test FAILED]megaease/easegress Pull Request 143 Deploy Test failed

**Review by megaeasex — APPROVED** [review]
[TASK:easegress-pr-test SUCCESS]megaease/easegress Pull Request 143 Deploy Test Success

**Review by benja-wu — APPROVED** [review]
LGTM

File: pkg/filter/wasmhost/hostfunc.go [hunk 1]
```diff
@@ -380,6 +382,14 @@ func (vm *WasmVM) hostLog(level int32, addr int32) {
 	}
 }
 
+func (vm *WasmVM) hostNow() int64 {
+	return time.Now().UnixNano() / 1e6
```
Comment (xxx7xxxx) [inline pkg/filter/wasmhost/hostfunc.go:386]: Why not just export nanosecond.

Comment (localvar) [reply pkg/filter/wasmhost/hostfunc.go:386]: there are two reasons:
1. The host functions need to support many languages, so we have to choose the lowest precision. 
2. Nanosecond requires 64 bits integers, but in some languages,  all numeric values are IEEE754 doubles and the max safe integer value is `2^53-1`, which is not larger enough to represent time in nanosecond (about 104 days).

Comment (xxx7xxxx) [reply pkg/filter/wasmhost/hostfunc.go:386]: The put the time unit into the function name of document.

Comment (localvar) [reply pkg/filter/wasmhost/hostfunc.go:391]: updated

**Review by xxx7xxxx — APPROVED** [review]

**Pull request merged**